src/configs/dev.config.ts

```diff
@@ -86,6 +86,21 @@ const devConfig: IConfig = {
             },
         },
     ],
+    badges: {
+        DISCORD_EMPLOYEE: "",
+        PARTNERED_SERVER_OWNER: "",
+        HYPESQUAD_EVENTS: "",
+        BUGHUNTER_LEVEL_1: "",
+        BUGHUNTER_LEVEL_2: "",
+        HOUSE_BRAVERY: "",
+        HOUSE_BRILLIANCE: "",
+        HOUSE_BALANCE: "",
+        EARLY_SUPPORTER: "",
+        TEAM_USER: "",
+        EARLY_VERIFIED_BOT_DEVELOPER: "",
+        DISCORD_CERTIFIED_MODERATOR: "",
+    },
```

Comment:
Discord bots have access to all emojis, provided they have the full ID. 

It would be advisable, rather than leaving these blank, to fill them with pre-existing emojis that represent these badges.

Any will do. If needed, I'll fill these in myself later. (PC access pending)

Reply:
I'll do it. I was sure that they don't have access to all emojis, and only to these from the servers they're in.

Reply:
It definitely was the case at one point, but I think it was updated along with webhook access to emojis some time ago.